scripts/release/detect-action-tiers.sh

```
@@ -0,0 +1,159 @@
+declare -A LATEST_TAGS
+declare -A HAS_CHANGES
+declare -A UNCOMMITTED
+
+for repo in "${ACTION_REPOS[@]}"; do
+  (
+    cd "actions/$repo" || exit 1
+    git fetch --tags -q
+    TAG=$(git describe --tags --abbrev=0 2>/dev/null || echo "NONE")
+    echo "TAG=$TAG"
+    if [ "$TAG" != "NONE" ]; then
+      CHANGES=$(git log "${TAG}..HEAD" --oneline -- action.yaml 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[info] Modification of CHANGES is local (to subshell caused by (..) group).

(SC2030)